org.openhab.binding.zigbee/README.md

```diff
@@ -153,23 +153,25 @@ The following coordinators are known to be supported.
 
 #### Ember EZSP NCP Coordinator
 
-The Ember EZSP NCP (Network Co-Processor) supports the SiLabs MightyGecko (EFR32MG) and the older EM357/8 dongles with the standard NCP firmware. The thing type is `coordinator_ember`. This dongle type is recommended due to its extensive use within both the community, and commercial systems using this dongle.
+The Ember EZSP NCP (Network Co-Processor) supports Silicon Labs Mighty Gecko (EFR32MG Family) and the older EM357/8 dongles with Silabs standard EmberZNet NCP Zigbee Coordinator firmware. The thing type is `coordinator_ember`. Ember dongles are recommended due to its extensive use within both the community and commercial systems using EmberZNet based hardware. Recommendation today is to get EFR32MG21 (EFR32 Mighty Gecko Series 2) or EFR32MG12 (EFR32 Mighty Gecko Series 1) based dongle as they support the latest EmberZNet NCP firmware.
 
 Note that there are generally two versions of the Ember NCP firmware in use. One operates at a baud rate of 115200 with RTS/CTS flow control (i.e. hardware flow control), the other operates at a baud rate of 57600, and XON/XOFF flow control (i.e. software flow control). If you are programming your own stick (e.g. the CEL stick) then it should be advisable to use the hardware flow control version - many commercial sticks seem to use the lower speed and software flow control (e.g. Bitron and Nortek HUSBZB-1).
 
-Firmware versions higher than 6.4.0.0 may be required to work with some newer devices (i.e. those conforming to the Zigbee 3.0 standard). If your dongle has older firmware then devices may appear to join the network, but will not work as they will leave the network if they do not receive updated security information.
+Firmware versions higher than 6.4.0.0 may be required to work with some newer devices (i.e. those conforming to the Zigbee 3.0 standard). If your dongle has older firmware then devices may appear to join the network, but will not work as they will leave the network if they do not receive updated security information. 
+
+EmberZNet Serial Protocol version 8 (EZSP v8) introduced in EmberZNet 6.7.x.x uses a new framing format and expanded command id field from 8 bits to 16 bits and is therefor not backward compatible. If you for whatever version want a backwards compatible firmware then use EmberZNet 6.6.x.x as it supports EZSP v5, v6 and v7 (so not EZSP v8).
```

Comment:
Please remove this line - it's not relevant for users what version of EZSP is used. This is user documentation and I would not recommend that people use old firmware.